client/source/DelphiLint.IDEContext.pas

```diff
@@ -359,7 +359,11 @@ procedure TToolsApiServices.ApplyTheme(Component: TComponent);
 
 procedure TToolsApiServices.RegisterFormClass(FormClass: TCustomFormClass);
 begin
+{$IF CompilerVersion < 33.0}
+  (BorlandIDEServices as IOTAIDEThemingServices250).RegisterFormClass(FormClass);
+{$ELSE}
   (BorlandIDEServices as IOTAIDEThemingServices).RegisterFormClass(FormClass);
+{$ENDIF}
```

Comment:
There's no need to do the ifdef here, just change to 250 if it's a compatibility issue. Newer Delphi versions also have the 250 version.